<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI Remote Controller</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="goButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>command Table</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>sensing Table</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="logText">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>411</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="sensingText">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>70</y>
      <width>491</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="leftButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="midButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rightButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>370</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>430</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>speech</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::SystemShutdown"/>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::MediaPlaybackPause"/>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QTextEdit" name="promptText">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>320</y>
      <width>491</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>300</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>openAI prompt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="entButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>460</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>goButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>go()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leftButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>left()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rightButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>right()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>405</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>434</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>midButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>mid()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>447</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>469</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>486</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>850</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>739</x>
     <y>479</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>930</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>930</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>entButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>enter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>510</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>go()</slot>
  <slot>mid()</slot>
  <slot>back()</slot>
  <slot>left()</slot>
  <slot>right()</slot>
  <slot>start()</slot>
  <slot>stop()</slot>
  <slot>enter()</slot>
 </slots>
</ui>
